systray.go

```diff
@@ -102,6 +108,10 @@ func RunWithExternalLoop(onReady, onExit func(), onAppearanceChanged func(bool))
 	}
 }
 
+func Show() {
+	showOnce()
```

Comment:
should this be?
```suggestion
	go showOnce()
```

Reply:
I don't think it needs to be, based on the example you have above